packages/flutter/test/widgets/form_test.dart

```diff
@@ -1457,4 +1457,39 @@ void main() {
 
     expect(focusNode2.hasFocus, isTrue);
   });
+
+  testWidgets('AutovalidateMode.always should validate on first build', (WidgetTester tester) async {
+    String errorText(String? value) => '$value/error';
+
+    Widget builder() {
+      return MaterialApp(
+        theme: ThemeData(),
+        home: Center(
```

Comment:
Looking great!

I noticed a couple of other small things:
- Setting theme data probably isn't necessary here.
- We could probably just call `pumpWidget` here, since this function isn't being referenced in multiple places.

```suggestion
    await tester.pumpWidget(
      MaterialApp(
        home: Center(
```
(ending punctuation would need to be adjusted as well.)